sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -908,7 +1046,7 @@ describe("rpc", () => {
         it(`run test: ${casename} (pwd=${opts.pwd},prog=${opts.program})`, asyncTest(async () => {
             // For each test case, run it twice: first to preview and then to update.
             for (const dryrun of [true, false]) {
-                console.log(dryrun ? "PREVIEW:" : "UPDATE:");
+                // console.log(dryrun ? "PREVIEW:" : "UPDATE:");
```

Comment:
Why was this commented out?

Reply:
Oh, it just generated lots of noise that i *personally* never found useful.  instead of a nice sequence of mocha results, we would get these two extra lines between everything.  I was trimming away things we didn't need here and commented that out.  I figured if someone needed this info, they could uncomment it.